### ISSUE TITLE
"ьЬь" inserted between each character when transliterating back to Russian

### ISSUE THREAD

**Issue by Stratus3D** [tjbladez/translit#13]
This commit (997899c68112edce325c8e1ac2768f8e2835ecfc) introduces a bug that causes "ьЬь" to be inserted between each character when transliterating back to Russian. The key with a value of "" is invalid. The key should be "'".

**Comment by Stratus3D** [comment]
This is fixed in https://github.com/euroteamoutreach/translit/pull/1

**Comment by tjbladez** [comment]
I will be happy to merge this in if you want to open up the pull request. We can version and republish the gem after that.

**Comment by Stratus3D** [comment]
Do you want my PR to include the rspec tests I wrote?

**Comment by tjbladez** [comment]
Absolutely, specs value is tremendous and overall I think specs are necessary to guide developer usage. I will convert existing `riot` specs to rspecs as well as part of that release.
